### ISSUE TITLE
using and opening tuunnelblick via menubar and keyboard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Tunnelblick/Tunnelblick#156]
```
using  voiceover I cannot click on tubbeelblick icon in òmenubar, it has no 
name, only a grafic symbol since you write tubbelblick in cocoa please make the 
same as for clock of volume icon  or do the same as in the installation 
procedure, this is perfect for blink users, also try using xcode and 64bit, it  
may help alot, manu of us ha  to work with openvpn under mac, thanks
```

Original issue reported on code.google.com by `migara...@gmail.com` on 31 Jul 2010 at 6:53
